plugins/MySQL.cpp

```diff
@@ -81,6 +81,33 @@ string MySQLPacket::serializeHandshake() {
     SAppend(handshake.payload, &connectionID, 4); // connection_id
     handshake.payload += (string) "xxxxxxxx";     // auth_plugin_data_part_1
     handshake.payload += lenEncInt(0);            // filler
+
+    static const unsigned CLIENT_LONG_PASSWORD = 0x00000001;
+    static const unsigned CLIENT_PLUGIN_AUTH   = 0x00080000;
+    const unsigned capability_flags = CLIENT_LONG_PASSWORD | CLIENT_PLUGIN_AUTH;
+
+    const unsigned short capability_flags_1 = (const unsigned short)(capability_flags);
+    const unsigned short capability_flags_2 = (const unsigned short)(capability_flags >> 16);
+    SAppend(handshake.payload, &capability_flags_1, 2); // capability_flags_1 (low 2 bytes)
```

Comment:
It looks like you are hand-rolling the encoding, which works, but is inconsistent with the rest of the file.  Please use `lenEncInt()` to be consistent with the rest of the file.